index.html

```diff
@@ -181,6 +181,30 @@ <h2 class="langFrameAndSkillsTitle">Skills</h2>
         </ul>
       </section>
     </article>
+    <article id="contact-form-container">
+      <header>
+        <h2 class="headline">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+      </header>
+      <section id="form-background">
+        <form id="contact-form" action="https://formspree.io/f/mzbwkgyl" method="POST">
+          <input class="input-fields" type="text" name="name" id="name" max="30" placeholder="Full Name" required>
```

Comment:
Great job! but I think you wanted to use the [maxlength](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/maxlength#examples), not the `max` attribute 😊

Please make sure to fix this.